src/sage/rings/polynomial/ideal.py

```diff
@@ -85,3 +85,18 @@ def groebner_basis(self, algorithm=None):
         gb = self.gens_reduced()
         from sage.rings.polynomial.multi_polynomial_sequence import PolynomialSequence_generic
         return PolynomialSequence_generic([gb], self.ring(), immutable=True)
+
+    def change_ring(self, R):
+        """
+        Coerce an ideal into a new ring.
+
+        EXAMPLES::
+
+            sage: P.<x,y> = LaurentPolynomialRing(QQ, 2)
+            sage: I = P.ideal([x + y])
+            sage: Q.<x,y,z> = LaurentPolynomialRing(QQ, 3)
+            sage: I.change_ring(Q)
+            Ideal (x + y) of Multivariate Laurent Polynomial Ring in x, y, z
+             over Rational Field
```

Comment:
These are not good tests as they are not testing the actual class defined here. If the `change_ring()` for the Laurent polynomial ideals is altered, then it might not actually test this method anymore.

Reply:
Concerning the failed tests, they are all successful in my machine.

Reply:
Should this go to the laurent_polynomial_ideal file?

Reply:
They can go there, yes.

Reply:
Again, you have not actually addressed my issue: This method is not actually being tested by your doctests. You need to explicitly create an ideal that tests this method; e.g.,
```
sage: R.<q> = QQ[]
sage: I = R.ideal([q^2+q-1])
sage: I.change_ring(RR['q'])
```